cross/fish/Makefile

```diff
@@ -0,0 +1,26 @@
+PKG_NAME = fish
+PKG_FULL_NAME = fish-shell
+PKG_EXT = tar.gz
+PKG_DOWNLOAD_METHOD = git
```

Comment:
official at the moment is quite outdated, with many bugs being fixed since then. soon I think there will be a brand new release. does it make sense to wait for next release and resubmit a pull request which uses the official tarball?